air-quality-ui/src/services/forecast-data-service.ts

```diff
@@ -0,0 +1,31 @@
+import { DateTime } from 'luxon'
+
+import { ForecastResponseDto } from './types'
+import { LocationType } from '../models'
+
+const API_URL = import.meta.env.VITE_AIR_QUALITY_API_URL
+
+export const getForecastData = async (
+  dateFrom: DateTime,
+  dateTo: DateTime,
+  baseTime: DateTime,
+  locationType: LocationType = 'city',
```

Comment:
Can we use the models.ts item here instead of the hardcoded 'city'?

Reply:
So the models.ts property is a literal type, it doesn't actually have a value. So what this is saying is that locationType must be of type 'LocationType'. The = part is setting a default value to be 'city' (which is the only permitted value via the type)